docs/developer-docs/latest/plugins/users-permissions.md

```diff
@@ -79,22 +79,29 @@ axios
 
 ### JWT configuration
 
-You can configure option for the JWT generation by creating `extensions/users-permissions/config/security.json` file.
+You can configure option for the JWT generation by using the plugin config `./config/plugins.js` file. See more information about this file in our [configuration](/developer-docs/latest/setup-deployment-guides/configurations/optional/plugins.md) section.
 We are using [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) to generate the JWT.
 
 Available options:
 
-- `expiresIn`: expressed in seconds or a string describing a time span zeit/ms.<br>
+- `jwtSecret`: random string used to create new JWTs, typically set using the `JWT_SECRET` environment variable.
```

Comment:
Do we need to add a line to the table in the [environment variables](https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.html#strapi-s-environment-variables) documentation, then?

Reply:
I was debating the same thing, this one is odd because by default we dump this to a .env file.

Probably wouldn't hurt